ansible/playbooks/tasks/azure-cluster-hana.yaml

```diff
@@ -23,21 +25,10 @@
     migration_threshold: "{{ (crm_conf_show.stdout | regex_search('migration-threshold=([0-9]*)', '\\1'))[0] }}"
   changed_when: false
 
-- name: Create HANA topology resource
+- name: Ensure maintenance mode is active
```

Comment:
This one looks like important and not related to Angi.

Reply:
Topology is still there, just diff is confused.

Reply:
Seems like change is:

- call one more `crm maintenance on` that was missing
- postpone `Create HANA topology resource` task execution 4 tasks later